### ISSUE TITLE
Nan output during training

### ISSUE THREAD

**Issue by hamiGH** [astorfi/3D-convolutional-speaker-recognition#24]
Hi @astorfi
In the development step, i tried running train_softmax.py over [VoxCeleb2 dataset](http://www.robots.ox.ac.uk/~vgg/data/voxceleb2/)  and then i got nan output (logits variable in code). how to solve this?

**Comment by astorfi** [comment]
@hamiGH It is hard to say as I do not know the details of your implementation. My guess is the learning rate must be adjusted. Furthermore, please make sure the batch-normalization is active.
